manifest.yaml

```diff
@@ -1,22 +1,10 @@
 #
 # We are using a GCE disk as persistent storage for the Hub
-# and eventually, each individual user node.
+# and for each individual user node.
 #
-# An administrator with access must provision these disks with a reasonable amount of space
-# before hand. Please note that with the K8s deployment, we are no longer using a shared NFS
-# and instead each user will have his or her own disk.
-#
-# Find the region for your cluster beforehand using:
-#
-#     gcloud container clusters list
-#
-# Provision your disk like so:
-#
-#     gcloud compute disks create hub-workdir-01 --size 1GiB
-#
-# Please make sure to delete your disk after you are done using it as you will be charged.
-#
-#     gcloud compute disks delete hub-workdir-01
+# An administrator no longer has to manually provision a disk for the Hub.
+# Since the pvc for the hub is using a storage class, the volume will automatically
+# be provisioned dynamically
```

Comment:
This comment makes sense given what was there before, but not as much if you read it on its own.

For example, someone reading it for the first time doesn't know what `no longer` is referring to.

Think you could describe exactly what gets provisioned at which points in time? Eg. When deploying, a disk for the hub gets provisioned. When starting a single user server, a disk for that user gets provisioned on the spot if it didn't exist before.